### PR TITLE
Test: use StdlibUnittest in BridgeNonVerbatim

### PR DESCRIPTION
**Pull request by dduan**
Convert tests in BridgeNonVerbatim.swift to use StdlibUnittest.

Also add a function that compares references in StdlibUnittest.

### CODE REVIEW

**Comment by dduan** [comment]
@swift-ci please test

**Comment by dduan** [comment]
@swift-ci please test OS X platform

**Comment by dduan** [comment]
@swift-ci please test OS X platform

File: test/stdlib/BridgeNonVerbatim.swift [hunk 1]
```diff
@@ -61,50 +61,48 @@ struct X : _ObjectiveCBridgeable {
...
+      expectEqual(one.value, 1)
+
+      // We can get the element again, but it may not have the same identity
+      let anotherOne = nsx.object(at: 0) as! LifetimeTracked
+      expectEqualReference(one, anotherOne)
 
-// CHECK-NEXT: leaks = 0 .
-print("leaks = \(LifetimeTracked.instances) .")
+      // Because the elements come back at +0, we really don't want to
+      // treat them as objects, or we'll get double deletion
```
Comment (dabrahams) [inline test/stdlib/BridgeNonVerbatim.swift]: Shouldn't you use `Unmanaged` here, then?

Comment (dduan) [inline test/stdlib/BridgeNonVerbatim.swift]: That's a good question for the original author: some Dave from 2014 :P

**Pull request merged**